plugins/processors/strings/README.md

```diff
@@ -78,6 +79,11 @@ If you'd like to apply multiple processings to the same `tag_key` or `field_key`
   ## Decode a base64 encoded utf-8 string
   # [[processors.strings.base64decode]]
   #   field = "message"
+
+  ## Sanitize a string to ensure it is a valid utf-8 string. Non-valid characters are replaced by 'new'
+  # [[processors.strings.valid_utf8]]
+  #   field = "message"
+  #   new = ""
```

Comment:
Could you change `new` to `replacement`? New does not describe what is happening in this code.